src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ContentServiceTests.cs

```diff
@@ -169,17 +172,317 @@ await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContex
             }
         }
 
+        [Fact]
+        public async Task AddComment()
+        {
+            var release = new Release
+            {
+                Content = new List<ReleaseContentSection>
+                {
+                    new ReleaseContentSection
+                    {
+                        ContentSection = new ContentSection
+                        {
+                            Heading = "New section",
+                            Order = 1,
+                            Content = new List<ContentBlock>
+                            {
+                                new HtmlBlock
+                                {
+                                    Created = new DateTime(2001, 1, 1),
+                                    Comments = new List<Comment>
```

Comment:
We also have an `AsList` method now in CollectionUtils that can make test code a bit more succinct when statically imported, like:

```
Comments = AsList(new Comment {Content = "Existing comment"})
```

rather than

```
Comments = new List<Comment>
{
    new Comment {Content = "Existing comment"}
}
```

Just so you're aware for the future.

Reply:
Cool - I've switched all instances of `new List` in ContentServiceTests to use `AsList`